testkit/package.json

```diff
@@ -31,13 +31,13 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "peerDependencies": {
-    "@kalix-io/kalix-javascript-sdk": "^1.0.0-M7"
+    "@kalix-io/kalix-javascript-sdk": "1.x"
```

Comment:
The expectation is for the SDK and testkit to be on the same version. The peer dependency on the SDK is simplified to just be `1.x`, as this would need to be manually updated — although not necessarily accurate if the testkit depends on newly introduced features in a minor version of the SDK. We could otherwise drop the peer dependency altogether.